ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-main.yaml

```diff
@@ -35,6 +50,13 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: metrics-linter
+      commands: /usr/bin/prom-metrics-linter
+      from: src
```

Comment:
After setting the image name above, use it instead of `src`.

Reply:
i.e.`metrics-linter`?

Reply:
Updated to `ssp-metrics-names-linter` (from line 20).